### PR TITLE
fix(header): fix header on small screen size

### PR DESCRIPTION
**Pull request by Aman-Codes**
## Description

Earlier on small screen size, the user icon dropdown was not displayed. Fixed it to now display dropdown on small screen as well.

### Changes

- Fixed header on small screen size

## Screenshots
image

image

## How to test

Test Header for screen size smaller than 991px.

### CODE REVIEW

File: src/components/Header/index.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+@media (max-width: 991px) {
```
Comment (Shruti3004) [inline src/components/Header/index.css:19]: @Aman-Codes I think we can add this css in global.css itself I don't think adding new file for it is worth.

File: src/components/Header/index.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  .dropleft .dropdown-menu {
+    right: unset !important;
+    left: unset !important;
```
Comment (Shruti3004) [inline src/components/Header/index.css:22]: @Aman-Codes if possible please add little bit padding as well.

**Review by Shruti3004 — CHANGES_REQUESTED** [review]

**Review by GMishx — APPROVED** [review]
Tested, working as expected.

File: src/components/Header/index.css (hunk 2 above)
Comment (Shruti3004) [reply src/components/Header/index.css]: @Aman-Codes please give padding as well.

Comment (Aman-Codes) [reply src/components/Header/index.css]: @Shruti3004 Updated it

File: src/styles/global.css [hunk 3]
```diff
@@ -102,3 +102,15 @@ h5,
...
+  .dropleft .dropdown-menu {
+    right: unset !important;
+    left: 3.5em !important;
+    padding: 1.5rem;
+  }
+}
+
+#basic-navbar-nav {
+  padding: 2rem;
+}
```
Comment (Shruti3004) [inline src/styles/global.css:116]: @Aman-Codes you should keep this class inside media query.

Screenshot from 2021-07-07 19-38-13

**Review by Shruti3004 — CHANGES_REQUESTED** [review]

**Review by Shruti3004 — APPROVED** [review]

**Pull request merged**